src/react/deckgl.js

```diff
@@ -114,7 +128,11 @@ export default class DeckGL extends React.Component {
       cachePosition: false,
       centerOrigin: false,
       onClick: this._onClick,
-      onMouseMove: this._onMouseMove
+      onMouseMove: this._onMouseMove,
+      onDragStart: this._onDragEvent,
+      onDragMove: this._onDragEvent,
+      onDragEnd: this._onDragEvent,
+      onDragCancel: this._onDragCancel
```

Comment:
Actually, digging deeper into this, we're _always_ running picking, even if no layers have `pickable` set. There's no check for existence of handlers or `pickable` truthiness before calling `addEvents`, we're just running picking on every mouse move and calling noops every frame.

Additionally, we currently have no facility for adding/removing event handlers after initialization; deck.gl calls luma's `addEvents` only once, on init; and `addEvents` itself does not support adding/removing handlers. We need to address this in the event handling refactor, @ibgreen.

All this indicates to me that:
- we should definitely fix this
- we should fix it in the event handling refactor, not in this PR.

If @ibgreen and @Pessimistress agree, I'd like to merge as-is.